modules/nf-core/amrfinderplus/run/tests/main.nf.test

```diff
@@ -0,0 +1,69 @@
+nextflow_process {
+
+    name "Test Process AMRFINDERPLUS_RUN"
+    script "../main.nf"
+    process "AMRFINDERPLUS_RUN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "amrfinderplus"
+    tag "amrfinderplus/run"
+    tag "amrfinderplus/update"
+
+    setup {
+
+        run("AMRFINDERPLUS_UPDATE") {
+            script "modules/nf-core/amrfinderplus/update/main.nf"
+            process {
+                """
+                """
+            }
+        }
+    }
+
+    test("amrfinderplus/run - haemophilus_influenzae - genome_fna_gz") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['haemophilus_influenzae']['genome']['genome_fna_gz'], checkIfExists: true)
```

Comment:
I think there may be a shift to using a new test data structure in the future, so maybe we can just update this module now! (https://nfcore.slack.com/archives/CJRH30T6V/p1706291621545369)
```suggestion
                    file(modules_testdata_base_path + 'genomics/prokaryotes/haemophilus_influenzae/genome/genome.fna.gz'], checkIfExists: true)
```